metacat-common/src/main/java/com/netflix/metacat/common/dto/HasBusinessMetadata.java

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package com.netflix.metacat.common.dto;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.netflix.metacat.common.QualifiedName;
+
+/**
+ * Marker interface for objects with business metadata.
+ */
+public interface HasBusinessMetadata extends HasMetadata {
```

Comment:
I realize this is an extension of existing pattern but this whole `has` as an interface name is weird to me. Like rather than implementing this you could build boolean methods into a higher level interface that just returns false by default or better yet the default `Metadata` interface has a `getBusinessMetadata` that just returns an `Optional<ObjectNode>`. Then if it doesn't exist it's an empty optional...